include/rocksdb/utilities/transaction_db.h

```diff
@@ -113,6 +113,28 @@ struct KeyLockInfo {
   bool exclusive;
 };
 
+struct DeadlockInfo {
+  TransactionID m_txn_id;
+  uint32_t m_cf_id;
+  std::string m_waiting_key;
+
+  DeadlockInfo() : m_txn_id(0), m_cf_id(0), m_waiting_key("0") {}
+  DeadlockInfo(TransactionID txn_id, uint32_t cf_id, std::string waiting_key)
```

Comment:
I don't think we need this either. The only place this is used, we can replace with list initialization:
http://en.cppreference.com/w/cpp/language/aggregate_initialization